bugsnag-android-core/src/main/java/com/bugsnag/android/EventInternal.kt

```diff
@@ -55,7 +54,7 @@ internal class EventInternal @JvmOverloads internal constructor(
     protected fun shouldDiscardClass(): Boolean {
         return when {
             errors.isEmpty() -> true
-            else -> errors.any { discardClasses.contains(it.errorClass) }
+            else -> errors.any { config.discardClasses.toHashSet().contains(it.errorClass) }
```

Comment:
This will create a new `HashSet` for each `Error` I don't think we want that overhead?

Reply:
True - I've reverted this change.